src/frontend/src/utils/status.ts

```diff
@@ -362,6 +362,16 @@ export const statusList = (pia: IPiaForm | null): StatusList => {
                 cancelLabel: 'Cancel',
               },
             },
+            {
+              status: 'COMPLETE',
+              modal: {
+                title: 'Submit for Completion',
+                description:
+                  'Once CPO has confirmed all necessary ministry reviews have occurred and data has been uploaded to the PID, PIA will move to “Complete” status.',
```

Comment:
Is the content same for Delegated flow as well, where CPO involvement is not there at all?

Reply:
confirm with UX just apply for delegate PIA, I think I just remove this from CPO list